packages/starlight/utils/navigation.ts

```diff
@@ -17,6 +22,7 @@ export interface Link {
 	href: string;
 	isCurrent: boolean;
 	badge: Badge | undefined;
+	attrs: LinkHTMLAttributes | undefined;
```

Comment:
If we have that default value in the schema, I guess this also would need updating:

```suggestion
	attrs: LinkHTMLAttributes;
```